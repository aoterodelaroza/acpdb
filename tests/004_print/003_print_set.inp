## check: 003_print_set.out -a1e-10
## labels: regression quick

verbose
connect ../zz_source/bsip2-small/bsip2.db
print set
