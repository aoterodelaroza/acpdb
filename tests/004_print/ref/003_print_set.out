* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file ../zz_source/bsip2-small/bsip2.db
Connected database is sane

* PRINT: print the contents of the database 

| id| key| litrefs| description|
| 1| s22| bsip1 boys1970 | A description.|

